#This is a synchronized stereo pair of rgb24 images
object_locator/SyncImageMsg images

#This is an 8 bit grayscale image that contains computed dispairty information
sensor_msgs/Image disparity_image
